testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/ra/AddConnectionFactoryToRAOnlineTest.java

```diff
@@ -0,0 +1,67 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import java.io.IOException;
+
+@RunWith(Arquillian.class)
+public class AddConnectionFactoryToRAOnlineTest {
+    private OnlineManagementClient client;
+    private Operations ops;
+    private Administration admin;
+    private static final String RA_ID = "genericjmsRA";
+
+    @Before
+    public void connect() throws IOException {
+        client = ManagementClient.online(OnlineOptions.standalone().localDefault().build());
+        ops = new Operations(client);
+        admin = new Administration(client);
+    }
+
+    @After
+    public void close() throws Exception {
+        ops.removeIfExists(Address.subsystem("resource-adapters").and("resource-adapter", RA_ID));
+        admin.reloadIfRequired();
+        client.close();
+    }
+
+    @Test
+    public void addConnectionFactoryTestcase() throws Exception {
+        client.apply(new AddResourceAdapter.Builder(RA_ID, "org.jboss.genericjms", TransactionType.NONE).build());
+        admin.reloadIfRequired();
+        client.apply(new AddConnectionFactoryToRA.Builder("cf1", "java:/jms/myRAConnectionFactory", RA_ID).build());
+        admin.reloadIfRequired();
+        ModelNodeResult res = ops.readResource(Address.subsystem("resource-adapters")
+                .and("resource-adapter", RA_ID)
+                .and("connection-definitions", "cf1"));
+        res.assertSuccess();
+    }
+
+    @Test
+    public void addConnectionFactoryWithGoodiesTestcase() throws Exception {
```

Comment:
Then please use something like "more detailed configuration" or "more parameters" or "detailed" or whatnot. There are no goodies in Creaper :-)